tests/Integration/Infrastructure/Controller/ListRegulationsControllerTest.php

```diff
@@ -14,15 +14,26 @@ public function testList(): void
         $crawler = $client->request('GET', '/');
 
         $this->assertResponseStatusCodeSame(200);
-        $this->assertSame('Restrictions de circulation', $crawler->filter('h3')->text());
+        $this->assertSame('Réglementations', $crawler->filter('h3')->text());
 
-        $firstTr = $crawler->filter('tbody > tr')->eq(0);
-        $secondTr = $crawler->filter('tbody > tr')->eq(1);
+        $tabs = $crawler->filter('.fr-tabs__list')->eq(0);
```

Comment:
Je pense qu'il vaut mieux mettre un `data-testid` explicite pour ne pas (trop) dépendre de détails d'implémentation du HTML

```suggestion
        $tabs = $crawler->filter('[data-testid="regulation-tabs"]')->eq(0);
```

Reply:
Hum, je ne suis pas trop fan du fait de rajouter du code dans le DOM juste pour pouvoir l'utiliser dans le cadre des tests.

Reply:
Je suis pas un intégriste du sujet, mais sur des projets précédents le couplage implicite pouvait donner de sacrés maux de tête en cas de refactoring : "Mince, qu'est-ce qu'on voulait cibler, déjà ?"

Donc, bon, ok

Surtout que, pour l'instant je trouve le WebTestCase assez limité, le besoin de cibler précisément en fonction de la structure HTML me rappelle Cypress. Sur Catalogage, on avait vraiment bien aimé le fait que Testing Library / Playwright permettent de faire du ciblage "sémantique" (en ciblant des "propriétés accessibles", ce qui permet de détecter des pb d'accessibilité sans tester toutes les balises, genre Testing Library sait qu'un bouton sans contenu ni `aria-label` n'aura pas de _name_)

Reply:
On en avait déjà parlé, mais ce serait peut-être le moment de se pencher sur https://github.com/symfony/panther ?

Reply:
Je re-relisais le README justement. Je crains que ça ne se base que sur des sélecteurs CSS, donc ça ne nous apportera pas tant que ça au niveau de la "résilience" des tests (https://kentcdodds.com/blog/making-your-ui-tests-resilient-to-change).

On va pouvoir cliquer sur des boutons par leur texte pour se balader, bref faire du E2E complet, mais on ne sera pas aussi aidé que TestingLibrary pour le UI unit testing, ou Playwright pour l'E2E.

Pour l'instant je n'ai pas l'impression qu'il y ait encore d'urgence à configurer Panther, car on n'a pas encore trop d'interactions JS personnalisés (hors DSFR) et autres comportements qu'on ne peut pas juste tester en inspectant la réponse HTML. Tu vois des assertions ici qu'on ne pourrait ajouter qu'avec Panther ?